AcadeMeData/test_msgboard.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+from AcadeMeData.models import MessageBoards
+
+
+@pytest.fixture
+def generate_msgboard(id=1, courseName="Linear Algebra"):
+    msgboard = MessageBoards(id=id, courseName=courseName)
+    msgboard.save()
+    return msgboard
+
+
+@pytest.mark.django_db
+class TestMessageBoardModel:
+    @pytest.fixture
+    def generate_msgboard(self, id=1, courseName="Linear Algebra"):
+        msgboard = MessageBoards(id=id, courseName=courseName)
```

Comment:
This is a fixture, should go out side

Where is the test of message boards? what do you want to test?

Reply:
For some reason the test code disappeared will fix..